ipatests/test_integration/test_ipa_cert_fix.py

```diff
@@ -94,3 +129,70 @@ def test_missing_csr(self, expire_cert_critical):
             else:
                 # timeout
                 raise AssertionError('Timeout: Failed to renew all the certs')
+
+    def test_renew_expired_cert_on_master(self, expire_cert_critical):
+        """Test if ipa-cert-fix renews expired certs
+
+        Test moves system date to expire certs. Then calls ipa-cert-fix
+        to renew them. This certs include subsystem, audit-signing,
+        OCSP signing, Dogtag HTTPS, IPA RA agent, LDAP and KDC certs.
+
+        related: https://pagure.io/freeipa/issue/7885
+        """
+        expire_cert_critical(self.master)
+
+        # wait for cert expiry
+        count = check_status(self.master, "8", "CA_UNREACHABLE")
+        assert count == 8
+
+        self.master.run_command(['ipa-cert-fix', '-v'], stdin_text='yes\n')
+
+        count = check_status(self.master, "9", "MONITORING")
+        # Count is 9 = 8 subsystem cert + 1 CA cert
+        assert count == 9
+
+        # second iteration of ipa-cert-fix
+        self.master.run_command(['ipa-cert-fix', '-v'], stdin_text='yes\n')
+        count = check_status(self.master, "9", "MONITORING")
+        # Count is 9 = 8 subsystem cert + 1 CA cert
+        assert count == 9
+
+    def test_ipa_cert_fix_non_ipa(self):
+        """Test ipa-cert-fix doesn't work on non ipa system
+
+        ipa-cert-fix tool should not work on non ipa system.
+
+        related: https://pagure.io/freeipa/issue/7885
+        """
+        result = self.master.run_command(['ipa-cert-fix', '-v'],
+                                         stdin_text='yes\n',
+                                         raiseonerr=False)
+        assert result.returncode != 0
```

Comment:
You can even check that resultcode = 2 (which corresponds to IPA not configured)